packages/components/carousel/src/index.ts

```diff
@@ -6,3 +6,4 @@ export { CarouselControlPrev, CarouselControlNext } from "./carousel-control"
 export type { CarouselControlProps } from "./carousel-control"
 export { CarouselIndicators } from "./carousel-indicators"
 export type { CarouselIndicatorsProps } from "./carousel-indicators"
+export type { CarouselControlRefType } from "./use-carousel"
```

Comment:
Please change `CarouselControlRefType` to `CarouselControl`.

Reply:
I understand!
I'll fix it!